lib/elixir_boilerplate_graphql/types/application.ex

```diff
@@ -0,0 +1,19 @@
+defmodule ElixirBoilerplateGraphQL.Types.Application do
+  use Absinthe.Schema.Notation
+
+  # Types
+  object :application_info do
```

Comment:
I think we should name the type like so:

```suggestion
  object :application do
```

No need for the `_info` suffix — this is "an application" we are describing here.